### PR TITLE
Write expenses policy.

### PR DESCRIPTION
**Pull request by drvinceknight**

### CODE REVIEW

File: source/policies/expenses-policy.rst [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+
+The approved HMRC travel rates for cars and motorcycles which can be
+found here:
+https://www.gov.uk/government/publications/rates-and-allowances-travel-mileage-and-fuel-allowances/travel-mileage-and-fuel-rates-and-allowances
+will be used when relevant.
+
+Claims procedure
+----------------
+
+Claims and associated documentation should be sent to the treasure of the UKPA,
```
Comment (doismellburning) [inline source/policies/expenses-policy.rst]: treasurer

File: source/policies/expenses-policy.rst [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+no later than 3 months afterr being incurred.
```
Comment (doismellburning) [inline source/policies/expenses-policy.rst]: after

Comment (drvinceknight) [reply source/policies/expenses-policy.rst]: :+1:

File: source/policies/expenses-policy.rst (hunk 1 above)
Comment (drvinceknight) [reply source/policies/expenses-policy.rst]: :+1:

**Pull request merged**